scripts/pdf/README.md

```diff
@@ -34,5 +34,7 @@ Complete information about the arguments can be viewed by running:
 The color-schemes that can be specified are:
 
```

Comment:
If you don't mind can you add images of the themes in README too? (Adding this for future reference, I think we can add the images to `images` directory in the repo instead of using the current one)